internal/conditions/error_to_message_test.go

```diff
@@ -0,0 +1,55 @@
+package conditions
+
+import (
+	"testing"
+
+	"github.com/kyma-project/telemetry-manager/internal/workloadstatus"
+)
+
+func TestErrorConverter(t *testing.T) {
```

Comment:
please create a test case that verifies it works with wrapped errors as well

Reply:
please add one with 
```err: fmt.Errorf("my super duper outer error: %w", &workloadstatus.PosIsNotScheduledError{})```

Reply:
added the test